docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,32 @@
+# Release 465 (7 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in EWKB files with the `ST_GeomFromBinary`
+  function. ({issue}`23824`) 
+* Add support for the `ORDER BY` clause in aggregate functions used with a
+  window context. ({issue}`23929`)
+* Add Accumulator and Group by hash CPU time metrics to verbose explain plans. ({issue}`21925`)
+
+## Faker connector 
+
+* Add the `random_string` catalog function to test Datafaker expressions for
+  random string creation. ({issue}`23990`)
+* Make generated data deterministic for repeated queries. ({issue}`24008`)
+
+## Hive connector
+
+* Add support to skip archiving when committing to a table in the Glue metastore
+  and the `hive.metastore.glue.skip-archive` configuration property is set to
+  `true`. ({issue}`23817`)
+* Use drop and create operation when replacing an existing SQL routine stored in
+  the Hive Metastore. ({issue}`24010`)
+
+## Iceberg connector
+
+* Add support for reading the `extra_properties` table property. ({issue}`17427`)
```

Comment:
What's the user visible impact? Where is it read? What functionality in Trino or the Iceberg connector is affected by this?

Reply:
Updated to use similar wording from when we added that feature for Hive tables.. just allows reading and writing arbitrary table properties